pkgs/tools/misc/tensorman/default.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgconfig, stdenv, rustPlatform, rustc, cargo, docker, openssl, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tensorman-${version}";
+  version = "0.1.0";
+
+ src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "tensorman";
+    rev = version;
+    sha256 = "0ywb53snvymmwh10hm6whckz7dwmpqa4rxiggd24y178jdfrm2ns";
+};
+buildInputs = [ pkgconfig openssl ];
+cargoSha256 = "1gh5w6zzrvjk60bqaf355fagijy723rvmqjh4laksd96pmzdfwn9";
```

Comment:
something with your indentation seems to be weird

Reply:
fixed